package.json

```diff
@@ -44,6 +44,6 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "cypress": "^3.1.0"
+    "cypress": "^2.1.0 || ^3.1.0 || ^4.0.0 || ^5.0.0"
```

Comment:
What about using `"cypress": "*"` ?

Reply:
Could be!
The only caveat I see on using `*` is that what if `cypress 6` comes along with a major breaking change? I know it's not a real concern today, but I think it's nice for a library to keep the dependencies list with only known working deps. That way, when cypress 6 comes out (or something like that), you or the core contributors can manually approve that this package is still compatible with the new version, and publish a new version...
But I can change if you want me to, no problem!

Reply:
👍 Im fine with that.